makefile

```diff
@@ -0,0 +1,22 @@
+filesIsPointer =  getAddresses.cpp identifier.cpp main.cpp
+filesScanProcesses = Scan.cpp main.cpp
+filesGetBaseAddresses = MemoryScan.cpp main.cpp
+filesRead = readAndGetContent.cpp main.cpp
+filesGetContent = readAndGetContent.cpp main.cpp
+rebulildables = main 
+ALL = getAddresses.cpp identifier.cpp Scan.cpp MemoryScan.cpp readAndGetContent.cpp main.cpp
```

Comment:
This needs to be updated every time we add a new file, right?
Can we not change this to ```*.cpp```?

Reply:
You mean for `ALL`, right?  Sounds good!

Reply:
Yes